syntaxes/d2.tmLanguage.json

```diff
@@ -173,7 +173,7 @@
         },
         {
           "name": "string.unquoted.d2",
-          "begin": "(?=[^[:space:]\\n#;\\[\\]{}|$'\"$])",
+          "begin": "(?=[^[:space:]\\n#;\\[\\]{}|$'\"])",
```

Comment:
are these changes intentional?

Reply:
Yes, @nhooyr suggested getting rid of the extra $.